StripeIdentity/StripeIdentityTests/Snapshot/IndividualWelcomeViewControllerSnapshotTest.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  IndividualWelcomeViewControllerSnapshotTest.swift
+//  StripeIdentityTests
+//
+//  Created by Chen Cen on 2/15/23.
+//
+
+import Foundation
+import iOSSnapshotTestCase
+
+@testable import StripeIdentity
+
+final class IndividualWelcomeViewControllerSnapshotTest: FBSnapshotTestCase {
+    static let mockVerificationPage = try! VerificationPageMock.response200.make()
+
+    override func setUp() {
+        super.setUp()
+        //        recordMode = true
```

Comment:
Nit: remove?

Reply:
we keep this line as is in case further changes are needed, [example](https://github.com/stripe/stripe-ios/blob/5c0e19ee3fe3b07b8080d293e4a1f487bf81bc7f/StripeIdentity/StripeIdentityTests/Snapshot/ListViewSnapshotTest.swift#L63)